.shippable.yml

```diff
@@ -34,7 +34,7 @@ build:
     - _make CFG_CRYPTO_{H,C,CBC_}MAC=n
     - _make CFG_CRYPTO_{G,C}CM=n
     - _make CFG_CRYPTO_{MD5,SHA{1,224,256,384,512,512_256}}=n
-    - _make CFG_WITH_PAGER=y
+    - _make CFG_WITH_PAGER=y out/arm-plat-vexpress/core/tee.bin out/arm-plat-vexpress/core/tee-pager.bin out/arm-plat-vexpress/core/tee-pageable.bin V=1
```

Comment:
1. The output path has no platform-specific path due to `O=out` (this is done to improve ccache efficiency)
2. Nitpicking: could be written as `out/core/{tee.bin,tee-pager.bin,tee-pageable.bin}` or even `out/core/tee{,-pager,-pageable}.bin`. And same below of course.

Reply:
I guess this applied to the TI case below too.